### PR TITLE
bump yarn env to 1.22.5 for theia-full and theia-cpp-docker

### PR DESCRIPTION
**Pull request by cyrilcros**
Closing https://github.com/theia-ide/theia-apps/issues/432
This updates the frozen YARN_VERSION used in a few Dockerfiles.
EDIT: also the copying/ chown’ing of /home/theia are now combined for theia-full; that should speed up build time.

### CODE REVIEW

**Comment by cyrilcros** [comment]
FYI 1.22.5 seems to fail, 1.21.0 may be okay when building the Dockerfile on my own computer. I will update the title / the push request once I figure it out.

**Comment by cyrilcros** [comment]
@vince-fugnitto : changing the theia-full Dockerfile to use 1.22.5 for yarn works up to a point on my computer (the Dockerfile has some issues after installing Dart). However, this near steps 48 of the build, whereas Travis fails at step 19 with the theia build process.
I notice that Travis runs most of the tests under Node 10 despite the NODE_VERSION environment variable using a node 12 version. Is that an issue?

**Comment by vince-fugnitto** [comment]
> I notice that Travis runs most of the tests under Node 10 despite the NODE_VERSION environment variable using a node 12 version. Is that an issue?

It's not, the node version used by travis is simply to execute the test suite (which is a basic sanity check to determine the application starts and has the proper views), and is separate than the actual images and building the application. I restarted the Travis build, `ripgrep` fails to download occasionally due to forks and not having the proper GitHub rate tokens.

**Comment by cyrilcros** [comment]
Thanks, the theia-full now ran to completion. Can you please rerun all the other failed tests? Thanks!

**Comment by cyrilcros** [comment]
@vince-fugnitto I now have the Docker image running at home ([Dockerhub link](https://hub.docker.com/repository/docker/ccros/theia-full/general)) and everything is fine. I got 2 last failing Travis CI tests, can you run them again? Thanks

**Comment by marcdumais-work** [comment]
Hi @cyrilcros 

> @vince-fugnitto I now have the Docker image running at home ([Dockerhub link](https://hub.docker.com/repository/docker/ccros/theia-full/general)) and everything is fine. I got 2 last failing Travis CI tests, can you run them again? Thanks

No worries - one of these always fails (needsto be removed) and you have not modified these images in your PR, in any case.

File: theia-full-docker/Dockerfile [hunk 1]
```diff
@@ -286,16 +286,16 @@ ENV PATH $DART_SDK/bin:/theia/.pub-cache/bin:$PATH
 
 WORKDIR /home/theia
 
-COPY --from=theia /home/theia /home/theia
+COPY --from=theia --chown=theia:theia /home/theia /home/theia
```
Comment (marcdumais-work) [inline theia-full-docker/Dockerfile:289]: nice touch

**Review by marcdumais-work — APPROVED** [review]
LGTM, thanks @cyrilcros . @vince-fugnitto any pending comment(s)?

**Comment by vince-fugnitto** [comment]
> LGTM, thanks @cyrilcros . @vince-fugnitto any pending comment(s)?

It looks fine to me :)

**Comment by marcdumais-work** [comment]
Thanks for this contribution, @cyrilcros. Merging.

**Pull request merged**